WTWSMS/events/religious_events.txt

```diff
@@ -30,7 +30,15 @@ province_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 9600
+		months = 8000
+		modifier = {
+			factor = 0.15
+			AND = {
+				NOT = { year = 500 }
+				religion = nicene # In the early parts of this Mod, the religious world was in turmoil with the fall of Christian WRE
```

Comment:
Are you sure on whataboutism? [Pravda would like to have a word with you. ](https://en.wikipedia.org/wiki/Whataboutism) As I said, ideally both would get it.